e2e/tabs.e2e-spec.ts

```diff
@@ -25,6 +36,18 @@ describe('Tabs', () => {
     });
   });
 
+  it('should match previous tabset screenshot when tab is hovered (screen: xs)',
+    (done) => {
+      SkyHostBrowser.get('visual/tabs');
+      SkyHostBrowser.setWindowBreakpoint('xs');
+      const tabElem = element(by.css('#screenshot-tabset button.sky-dropdown-button-type-tab'));
+      browser.actions().mouseMove(tabElem).perform();
+      expect('#screenshot-tabset').toMatchBaselineScreenshot(done, {
+        screenshotName: 'tabset-xs-hover'
+      });
+      SkyHostBrowser.moveCursorOffScreen();
```

Comment:
Was this a necessary line (moving the cursor off screen at the end) and should it be in the lg screen test too?

Reply:
It is more of a sanity check since the test in deliberately moving the mouse onto the screen. I wanted to ensure that we were not potentially triggering any unwanted states in future tests. It probably should be added to the large test. I'll do that. I don't think we need it in the other tests as they aren't moving the mouse onto the screen.

Reply:
Done